tests/unit/test-class-wc-payment-gateway-wcpay.php

```diff
@@ -112,8 +112,6 @@ public function tearDown() {
 		// Fall back to an US store.
 		update_option( 'woocommerce_store_postcode', '94110' );
 		$this->wcpay_gateway->update_option( 'saved_cards', 'yes' );
-
-		delete_option( '_wcpay_feature_grouped_settings' );
```

Comment:
There's another test case referencing the `_wcpay_feature_grouped_settings` flag, can the `update_option( '_wcpay_feature_grouped_settings', '1' );` be removed as well?